src/core/border_router/routing_manager.cpp

```diff
@@ -757,6 +757,9 @@ void RoutingManager::SendRouterAdvertisement(const Ip6::Prefix *aNewOmrPrefixes,
         if (error == kErrorNone)
         {
             otLogInfoBr("sent Router Advertisement on interface %u", mInfraIfIndex);
+#if OPENTHREAD_CONFIG_REFERENCE_DEVICE_ENABLE
+            otDumpCertBr("[BR-CERT] direction=send | type=RA |", buffer, bufferLength);
```

Comment:
```suggestion
            otDumpCertBr("[BR-CERT] direction=sent | type=RA |", buffer, bufferLength);
```

Reply:
The commissioner cert log uses the word "send", following it to avoid parsing issues:
https://github.com/openthread/openthread/blob/5b6654d0bccaad436193a08a79a545e8a22b64a0/src/core/meshcop/commissioner.cpp#L1133